sklearn/setup.py

```diff
@@ -78,7 +79,8 @@ def configuration(parent_package='', top_path=None):
     # add the test directory
     config.add_subpackage('tests')
 
-    maybe_cythonize_extensions(top_path, config)
+    if 'sdist' not in sys.argv:
+        cythonize_extensions(top_path, config)
```

Comment:
I am not too sure about this as it prevents to do things like `python setup.py sdist bdist_wheel` in a single line but maybe we don't care. Our CI does not do this.